templates/generic/entry-type/tests/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{kebab_case entry_type.name}}.test.ts.hbs

```diff
@@ -24,8 +24,8 @@ test('create {{pascal_case entry_type.name}}', async () => {
     // This assumes app bundle created by the `hc app pack` command.
     const testAppPath = process.cwd() + '/../workdir/{{app_name}}.happ';
 
-    // Set up the app to be installed 
-    const appSource = { appBundleSource: { path: testAppPath } };
+    // Set up the app to be installed
+    const appSource = { appBundleSource: { type: "path", value: testAppPath } as AppBundleSource };
```

Comment:
The type assertion was added, so I suppose it produced an error without it. That's lame. Does this work?
```suggestion
    const appBundleSource: AppBundleSource = { type: "path" value: testAppPath };
    const appSource = { appBundleSource };
```

Reply:
Yea, the ts compiler will complain about these. Updated though